### DISCUSSION TITLE
【質問】ユーザ指定でのサイト内のあいまい検索はどうすれば良いでしょうか？？

### DISCUSSION THREAD

**Discussion by kouji0705** [increments/qiita-discussions#566] [ヘルプ] [answered]
自身や他ユーザの過去投稿より、検索ワードであいまい検索を実施したい。
どのように実施すれば良いでしょうか？？下記画像の赤枠から実施できますでしょうか？？
<img width="1398" alt="スクリーンショット 2023-11-19 16 21 27" src="https://github.com/increments/qiita-discussions/assets/45913424/b6cf6fae-37df-4283-b39d-33706f257c5b">

**Comment by tanaka-asami415** [comment, marked as answer]
@kouji0705 
検索オプション（`user:ユーザー名`）で検索の対象ユーザーを指定していただくことが可能です。

検索オプションについては、検索結果ページに表示される検索バー横のアイコンをクリックしていただくか
![image](https://github.com/increments/qiita-discussions/assets/103916185/cc39e517-1040-429f-8b97-ab566589b71a)


ヘルプページにてご確認いただけます！✨
https://help.qiita.com/ja/articles/qiita-search-options
